libcxx/test/std/localization/codecvt_unicode.pass.cpp

```diff
@@ -618,7 +618,7 @@ template <class InternT, class ExternT>
 void utf8_to_utf16_in_error(const std::codecvt<InternT, ExternT, mbstate_t>& cvt) {
   // UTF-8 string of 1-byte CP, 2-byte CP, 3-byte CP, 4-byte CP
   const unsigned char input[] = "b\u0448\uD700\U0010AAAA";
-  const char16_t expected[]   = {'b', 0x0448, 0xD700, 0xDBEA, 0xDEAA, 0};
+  const InternT expected[]    = {0x62, 0x0448, 0xD700, 0xDBEA, 0xDEAA, 0};
```

Comment:
It seems to me that these tests are using the API as it was intended (before it was removed for being bad)?
As in, they are using something like `std::codecvt_utf8_to_utf16` with the char16_t type in their code.

And this set of changes removes test coverage for that case, because it generates the warning about potential narrowing?

Reply:
Also, the point of having the literal `b` in the expected output is to match the `b` in the input,
which gets totally lost with this change.

Would `static_cast<InternT>('b')` work instead?

On second thought, why is the change from `b` to 0x62 needed ? That will never narrow or change meaning, right?

Reply:
These changes ('b) were a no-op, I reverted them